vtex-sites/storecomponents.store
# Add toast to layout component

## What's the purpose of this pull request?
To use the optimistic order form a toast component is required.

## How it works? 
When orderForm context has messages it uses a toast to show the messages.

## How to test it?
Go to store ([preview below](https://sfj-604205d--storecomponents.vtex.app/))
1.  click to add `Jobs top notebook`;
2. Add 1000 of this item;
3. A toast will appear after the request to update the item quantity
